### ISSUE TITLE
How to train attmodel?

### ISSUE THREAD

**Issue by LaiPiXiong** [ruotianluo/ImageCaptioning.pytorch#66]
I extract fc_feat and att_feat from resnet101 (Not download from your link).  BUT, it not works well. The CIDER score is very low. Does any skill there to train att2in2 model?

**Comment by ruotianluo** [comment]
Are you training on coco?

**Comment by LaiPiXiong** [comment]
No, I use pretrained resnet200 in MSCOCO.

**Comment by LaiPiXiong** [comment]
And now, the train loss is lower than 0.25, but the score is only round 0.69. Is this normal?

**Comment by ruotianluo** [comment]
It looks wierd. Usually the loss would go to 2.x.

**Comment by LaiPiXiong** [comment]
Oh, yes it's 2.x. 
![image](https://user-images.githubusercontent.com/15033269/45594710-ba516500-b9d1-11e8-83b7-afa32c693838.png)

**Comment by LaiPiXiong** [comment]
And CIDER is as below,
![image](https://user-images.githubusercontent.com/15033269/45594725-ebca3080-b9d1-11e8-9987-d2fec1969413.png)

**Comment by ruotianluo** [comment]
Att2in2 should be much better.

**Comment by LaiPiXiong** [comment]
OK, I see, thank you very much, and I will try it again.
